plugins/search/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -106,6 +115,68 @@ const CheckboxFilter = ({
   );
 };
 
+const TypeFilter = ({
```

Comment:
Would consider renaming to `SearchTypeFilter` to fit with the other exports?  But also, maybe even name it something different / move it to its own folder, since it's operating on a different part of the search context than the rest of the filters here.  Maybe even just `<SearchType />`?

You could imagine variants in the future...  `<SearchType.Menu />` or `<SearchType.Tabs />`, where the default is the equivalent of what you have here `<SearchType.Select />`

Reply:
Yeah, I wasn't quite sure about this. I think it makes sense both ways since realistically _it is_ a type of filter, but then again within the actual codebase it isn't. I'll move it over to be self contained to keep the current filter filter dedicated to those only.